framework/db/mssql/QueryBuilder.php

```diff
@@ -35,7 +35,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
         Schema::TYPE_BIGINT => 'bigint',
         Schema::TYPE_FLOAT => 'float',
         Schema::TYPE_DOUBLE => 'float',
-        Schema::TYPE_DECIMAL => 'decimal',
+        Schema::TYPE_DECIMAL => 'decimal(18,0)',
```

Comment:
Maybe (5,2) is a good value, because (10,0) is similar to INT(10), don't have decimal